src/myr/Myr.js

```diff
@@ -962,11 +1012,44 @@ class Myr {
       loop: ${Boolean(this.loop)};
       isRawProperty: true;
       type: color;
+      startEvents: ${this.startEvent};
+      delay: ${this.delay};
     `;
         el.animation__color = anim;
         return outerElId;
     }
 
+    setDelay = (ts) => {
+        if (typeof ts === "number") {
+            this.delay = ts;
+        } else {
+            console.error("must pass a numeric for setDelay");
+        }
+        return this.delay;
+    };
+
+    events = ["click", "mouseenter", "mouseleave", "none"];
+
+    setAnimationTrigger = (event = "none") => {
+
+        if (typeof event === "string") {
+            event = event.toLowerCase();
+            if (this.events.includes(event)) {
+                if (event !== "none") {
```

Comment:
Looking at this again, this should probably be the opposite. `if(event === "none")`, then do the special case. Else do the standard case where `this.startEvent = event`.